submissions/LGeorge007/hooli-style-popup/index.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hooli style popup</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <nav class="nav-panel">
+            <ul class="nav-list">
+                <li class="nav-item"><a href="#" class="mail">Mail</a></li>
+                <li class="nav-item"><a href="#" class="img">Images</a></li>
+                <li class="nav-item">
+                    <div class="popup-wrapper">
+                        <input type="checkbox" id="popup-button" class="popup-input">
```

Comment:
`id = "popup-button"` - this selector was added what for?  could you please explain?

Reply:
I used input type=checkbox with id = "popup-button".
And use id for binding checkbox and its label.

Reply:
And yes. I have not used the following construction:
<label>
          <input type = "checkbox">
</label>

I used the standard spelling:
<input type = "checkbox" id = "popup-button">
<label for = "popup-button">.

To hide the popup-input, I use -- appearance: none;